lib/signed-xml.js

```diff
@@ -891,7 +892,14 @@ SignedXml.prototype.createReferences = function(doc, prefix) {
 
         var trans = ref.transforms[t]
         var transform = this.findCanonicalizationAlgorithm(trans)
-        res += "<" + prefix + "Transform Algorithm=\"" + transform.getAlgorithmName() + "\" />"
+        res += "<" + prefix + "Transform Algorithm=\"" + transform.getAlgorithmName() + "\""
+        if (ref.inclusiveNamespacesPrefixList && ref.inclusiveNamespacesPrefixList.length > 0) {
```

Comment:
You check to see if `.length > 0`, but I think you mean to test `.length === 1` because I see no loop to handle anything other than a length of 1.

Reply:
The property `inclusiveNamespacesPrefixList` is a string, not an array. But you are correct, the length check is not needed at all.